README.md

```diff
@@ -19,72 +23,92 @@ yarn create testcafe
 pnpm create testcafe
 ```
 
-## Options
-
-The `create-testcafe` tool accepts the following options:
-
-### run-wizard
+## Specify project name and location
 
-If you enable the `--run-wizard` option, the `create-testcafe` tool will display an interactive wizard that allows you to specify project settings:
+To specify the name and location of the target folder, pass a valid path (relative or absolute) to the `create-testcafe` command:
 
-**Type**: Boolean
-
-**Default value**: `true` unless the project directory already includes any project.
-
-**Alias**: `-w`
+```sh
+# npx
+npx create-testcafe appName
+# npm 
+# Preface the argument with a double dash
+npm init testcafe@latest -- appName
```

Comment:
It is not necessary to add a double dash if only the path parameter is specified.